### DISCUSSION TITLE
Any plans to add Graphs like in the closed source Fan Control?

### DISCUSSION THREAD

**Discussion by xXRealXxx** [wiiznokes/fan-control#92] [General]
I would like to be able to set up my case fans to spin up depending on the temperature of both my CPU and GPU via those easy to understand Graphs.

**Comment by wiiznokes** [comment]
Yes, it is planned. Not a priority tho but it will eventually be supported. The hard part will be the UI because AFAIK, there is no one that made a modifiable graph with Iced, but i could easily implement it with config file

**Comment by xXRealXxx** [comment]
> Yes, it is planned. Not a priority tho but it will eventually be supported. The hard part will be the UI because AFAIK, there is no one that made a modifiable graph with Iced, but i could easily implement it with config file

Oh, that's nice!

**Comment by wiiznokes** [comment]
It's implemented by #93. Once [ci](https://github.com/wiiznokes/fan-control/actions/runs/8589679974) is finish, you can test via [nightly release](https://github.com/wiiznokes/fan-control/releases) ;)

There is some know problem like dark mode for the window to add coordinates
